libraries/botbuilder-dialogs/tests/test_activity_prompt.py

```diff
@@ -2,35 +2,203 @@
 # Licensed under the MIT License.
 
 import aiounittest
-from botbuilder.dialogs.prompts import ActivityPrompt, NumberPrompt, PromptOptions, PromptRecognizerResult
-from botbuilder.schema import Activity, InputHints
+import unittest
 
-from botbuilder.core.turn_context import TurnContext
+from typing import Callable
+from botbuilder.dialogs.prompts import (ActivityPrompt, NumberPrompt, PromptOptions, PromptRecognizerResult,
+                                        PromptValidatorContext)
+from botbuilder.schema import Activity, InputHints, ActivityTypes
+
+from botbuilder.core import ConversationState, MemoryStorage, TurnContext, MessageFactory
 from botbuilder.core.adapters import TestAdapter
+from botbuilder.dialogs import DialogSet, DialogTurnStatus, DialogReason
+
+
+async def validator(prompt_context: PromptValidatorContext):
+    tester = unittest.TestCase()
+    tester.assertTrue(prompt_context.attempt_count > 0)
+
+    activity = prompt_context.recognized.value
+
+    if activity.type == ActivityTypes.event:
+        if int(activity.value) == 2:
+            prompt_context.recognized.value = MessageFactory.text(str(activity.value))
+            return True
+    else:
+        await prompt_context.context.send_activity("Please send an 'event'-type Activity with a value of 2.")
+
+    return False
+
 
 class SimpleActivityPrompt(ActivityPrompt):
-    pass
+    def __init__(self, dialog_id: str, validator: Callable[[PromptValidatorContext], bool]):
+        super().__init__(dialog_id, validator)
+
 
 class ActivityPromptTests(aiounittest.AsyncTestCase):
     async def test_does_the_things(self):
```

Comment:
We should probably actually just remove this test case-- I left it in there, but didn't get around to putting any assertions or anything in there

Reply:
referencing test called `async def test_does_the_things()`

Reply:
Solved (: